oshdb-api/src/test/java/org/heigit/ohsome/oshdb/api/tests/TestOhsomeFilter.java

```diff
@@ -79,12 +78,20 @@ public void testAggregateFilter() throws Exception {
 
   @Test
   public void testAggregateFilterObject() throws Exception {
-    MapReducer<OSMEntitySnapshot> mr = createMapReducer();
-    SortedMap<OSMType, Integer> result = mr
+    SortedMap<OSMType, Integer> result = createMapReducer()
         .aggregateBy(x -> x.getEntity().getType())
         .filter(filterParser.parse("(geometry:polygon or geometry:other) and building=*"))
         .count();
 
     assertEquals(42, result.get(OSMType.WAY).intValue());
   }
+
+  @Test
+  public void testFilterGroupByEntity() throws Exception {
+    MapReducer<OSMEntitySnapshot> mr = createMapReducer();
+    Number osmTypeFilterResult = mr.groupByEntity().osmType(OSMType.WAY).count();
```

Comment:
`osmType(…)` is deprecated. What would be the plan for this test, if the method is removed in the future? Maybe add a comment to this test explaining what needs to be done if this method is removed.

Reply:
right. that would be one of the todos in #324